routes/merge.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const fs = require('fs');
+
+
+
+/* GET merge colors. */
+router.get('/', function(req, res) {
+  let schemeMap = req.app.locals.scheme;
+  let colorsMap = req.app.locals.colors;
+  let mergeTo = req.query.mergeTo;
+  let mergedColors = req.query.colors;
+
+  mergeColorsHandler(colorsMap, mergeTo, mergedColors);
+
+  res.status(200).send('Merged!');
+});
+
+function mergeColorsHandler(colorsMap, mergeTo, mergedColors) {
+  let colors = mergedColors.split(';');
+  console.log(colors);
+
+  for (let i in colors) {
+    let color = colors[i];
+    let colorObj = colorsMap.get(color);
+    let colorDataMap = groupColorDataByFile(colorObj.meta);
```

Comment:
Colors should be grouped by files, though one file can contain more than one color that will be replaced by mergeTo argument.